glfw/ibus_glfw.c

```diff
@@ -388,7 +388,7 @@ ibus_key_state(unsigned int glfw_modifiers, int action) {
     M(CONTROL, IBUS_CONTROL_MASK);
     M(ALT, IBUS_MOD1_MASK);
     M(NUM_LOCK, IBUS_MOD2_MASK);
-    M(SUPER, IBUS_MOD4_MASK);
```

Comment:
Why is IBUS_MOD4_MASK no longer mapped to super?

Reply:
That is an embarrassing bug. Thank you for catching it. Fix pushed.

Reply:
@kovidgoyal Shouldn't there be also

```
M(META, IBUS_MOD1_MASK);
M(HYPER, IBUS_MOD3_MASK);
```

Reply:
No idea somebody that uses ibus and hyper/meta needs to test and validate.